src/vdaf/suite.rs

```diff
@@ -6,7 +6,7 @@
 //
```

Comment:
Changes in this file will collide with https://github.com/abetterinternet/libprio-rs/pull/181

Reply:
It looks like `suite.rs` is removed by that PR, but it looks like trivial updates will be require to the `prg.rs` that replaces it (a `Decode` implementation will need to drop its `<()>` parameterization & unused `_decoding_parameter` parameter).

Want me to wait for that PR to make it in to merge this PR? I'd like to get this merged sooner rather than later to unblock work on `janus` but I think I could do some local development by overriding the crate location in my local `cargo.toml`.

Reply:
Your call. I think Chris is likely to merge his change later today.

Reply:
I waited, the merge wasn't too bad & probably easier for me to handle than to foist off on Chris.